h2o-genmodel/src/main/java/hex/genmodel/easy/error/CountingErrorConsumer.java

```diff
@@ -13,24 +13,57 @@
  */
 public class CountingErrorConsumer extends EasyPredictModelWrapper.ErrorConsumer {
 
-  private AtomicLong dataTransformationErrorsCount = new AtomicLong();
-  private Map<String, AtomicLong> unknownCategoricalsPerColumn = new ConcurrentHashMap<>();
+  private Map<String, AtomicLong> dataTransformationErrorsCountPerColumn;
+  private Map<String, AtomicLong> unknownCategoricalsPerColumn;
 
   /**
    * @param model An instance of {@link GenModel}
    */
   public CountingErrorConsumer(GenModel model) {
+    initializeDataTransformationErrorsCount(model);
+    initializeUnknownCategoricalsPerColumn(model);
+  }
+
+  /**
+   * Initializes the map of data transformation errors for each column that is not related to response variable,
+   * excluding response column. The map is initialized as unmodifiable and thread-safe.
+   *
+   * @param model {@link GenModel} the data trasnformation errors count map is initialized for
+   */
+  private void initializeDataTransformationErrorsCount(GenModel model) {
+    String responseColumnName = model.isSupervised() ? model.getResponseName() : null;
+
+    dataTransformationErrorsCountPerColumn = new ConcurrentHashMap<>();
+    for (String column : model.getNames()) {
+      // Do not perform check for response column if the model is unsupervised
+      if (!model.isSupervised() || !responseColumnName.equals(column)) {
```

Comment:
I would rather do `column.equals(responseColumnName)` - rather be safe and account for the case when responseColumnName would be null

Reply:
Sure.